src/hotspot/share/runtime/handshake.cpp

```diff
@@ -327,37 +290,36 @@ void HandshakeOperation::do_handshake(JavaThread* thread) {
   // It is no longer safe to refer to 'this' as the VMThread/Handshaker may have destroyed this operation
 }
 
-void Handshake::execute(HandshakeClosure* thread_cl) {
-  HandshakeOperation cto(thread_cl);
+void Handshake::execute(HandshakeClosure* hs_cl) {
+  HandshakeOperation cto(hs_cl, NULL);
   VM_HandshakeAllThreads handshake(&cto);
   VMThread::execute(&handshake);
 }
 
-bool Handshake::execute(HandshakeClosure* thread_cl, JavaThread* target) {
```

Comment:
As I said in PR request:
"Removed VM_HandshakeOneThread, the VM thread uses the same call path as direct handshakes did."
They are not VM ops, and default we do not promise who will execute it.
Just that it will be executed on thread X.